src/Service.Tests/Unittests/SerializationDeserializationTests.cs

```diff
@@ -422,7 +422,7 @@ private static void VerifyParameterDefinitionSerializationDeserialization(Parame
         {
             // test number of properties/fields defined in Column Definition
             int fields = typeof(ParameterDefinition).GetFields(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance).Length;
-            Assert.AreEqual(fields, 4);
+            Assert.AreEqual(fields, 5);
```

Comment:
magic number. not sure why this value increased from 4 to 5. I understand that this was existing test, but none of the comments surrounding this code explain the number.

Reply:
new field in `ColumnDefinition` : `public SqlDbType? SqlDbType { get; set; }`